src/utils/api/delegate.js

```diff
@@ -19,15 +19,25 @@ export const vote = (activePeer, secret, publicKey, voteList, unvoteList, second
   });
 
 export const getVotes = (activePeer, address) =>
-  requestToActivePeer(activePeer, 'accounts/delegates/', { address });
+  activePeer.votes.get({ address });
 
 export const getVoters = (activePeer, publicKey) =>
-  requestToActivePeer(activePeer, 'delegates/voters', { publicKey });
+  activePeer.voters.get({ publicKey });
 
-export const registerDelegate = (activePeer, username, secret, secondSecret = null) => {
-  const data = { username, secret };
-  if (secondSecret) {
-    data.secondSecret = secondSecret;
+export const registerDelegate = (activePeer, username, passphrase, secondPassphrase = null) => {
```

Comment:
Is registerDelegate part of the same ticket ?

Reply:
It shouldn't be, there is a separate PR for that https://github.com/LiskHQ/lisk-hub/pull/972 but I see that some commits are in both PRs